src/graph_export/BUILD

```diff
@@ -0,0 +1,66 @@
+#
+# Copyright (c) 2025 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+load("//:common_settings.bzl",
+     "COMMON_STATIC_LIBS_COPTS", "COMMON_STATIC_LIBS_LINKOPTS", "COMMON_FUZZER_COPTS", "COMMON_FUZZER_LINKOPTS", "COMMON_LOCAL_DEFINES", "PYBIND_DEPS")
+
+COPTS_ADJUSTED = COMMON_STATIC_LIBS_COPTS + select({
+        "//conditions:default": [],
+        "//:fuzzer_build" : COMMON_FUZZER_COPTS,
+})
+
+LINKOPTS_ADJUSTED = COMMON_STATIC_LIBS_LINKOPTS + select({
+    "//conditions:default": [],
+    "//:fuzzer_build" : COMMON_FUZZER_LINKOPTS,
+})
+
+cc_library(
+    name = "graph_export",
+    srcs = ["graph_export.cpp"],
+    hdrs = ["graph_export.hpp"],
+    deps =  [
+        "@ovms//src:cpp_headers",
+        "@ovms//src:libovmsstring_utils",
+        "@ovms//src:libovmslogging",
+        "@ovms//src:libovms_server_settings",
+        "@ovms//src:libovms_module",
+        "@ovms//src:libovmsfilesystem",
```

Comment:
Why @ovms prefix? its slightly less visible what is comming from external deps IMO.

Reply:
This is good practice when someone wants to compile ovms as dependency than he does not need to add those references to ovms.